slick.groupitemmetadataprovider.js

```diff
@@ -42,8 +42,18 @@
       includeHeaderTotals: false
     };
 
-    options = $.extend(true, {}, _defaults, options);
```

Comment:
now thinking about it, the fact that you removed this options copy will break most everyone's code. You have to set the options somewhere, so now unless the user is calling `setOptions` he won't have any options (not even the defaults). I suggest that you add this extend at least in the `init()` method.

Reply:
please check [line 47](https://github.com/6pac/SlickGrid/blob/778b1e6460f56ccda01fd7db07bda56ae66bf695/slick.groupitemmetadataprovider.js#L47). the function is called once per constructor, so no need to call it in init function...

Reply:
ok I see but I would still prefer the changes that I mentioned for the `init` and `setOptions` which is what all other plugins/controls are doing. I don't think we should extend the defaults when calling `setOptions`.

Reply:
please check https://github.com/6pac/SlickGrid/pull/466#discussion_r382673606